docs/docsite/rst/inventory_guide/intro_inventory.rst

```diff
@@ -262,6 +262,11 @@ You can target this inventory directory as follows:
 
 You can also configure the inventory directory in your ``ansible.cfg`` file. See :ref:`intro_configuration` for more details.
 
+Inventory files are also read from the sub-directories of a specified
+inventory directory, but sub-directories are not searched for
+``host_vars`` or ``group_vars`` :ref:`directories
+<splitting_out_vars>`.  Sub-directories with these names are ignored.
+
```

Comment:
This is partially incorrect and due to not distinguishing what inventory sources are from what vars plugins provide.

All vars plugins (host_group_vars vars plugin governs the inclusion of host_vars/group_vars dirs) use the 'inventory source' as a base, when that is a directory the multiple files/subdirs do not count as independent inventory sources, just the original directory does.  

Those directories (and `vars_plugins`) are ignored if present, if not they would not have the same function you seem to expect. They would be processed at as part of an inventory source and they are probed as  containing inventory files, but not vars files, which would end up either with warnings, errors or worse, creating unexpected hosts and groups depending on enabled inventory plugins.

Reply:
Thank you for your review.  I've pushed a patch in response.

The new docs do not include mention of INVENTORY_IGNORE_PATTERS, etc.  That seems overkill here.

Reply:
I added links to the ignore config settings, just to be through.  Hopefully this is not confusing.

Reply:
I found so many things that needed updating in this page that i ended up creating my own PR #2416